packages/app/src/pages/endpoints/[endpointId]/_/content/types/table/index.tsx

```diff
@@ -104,6 +112,18 @@ const ContentTable: React.FC<Props> = ({
     [document, base, sorts]
   );
 
+  const [rowObject, setRowObject] = useState<Data>();
+  const onRowClick = useCallback(
```

Comment:
We often use not on but handle for click handler.

Reply:
I will use `handle`  for a click handler.